### PR TITLE
Fix failing tax step test in release branch

### PR DESCRIPTION
**Pull request by sangeetha-nayak**

### CODE REVIEW

File: tests/cypress/integration/Home/homePageWithWoo.cy.js [hunk 1]
```diff
@@ -186,8 +196,13 @@ describe( 'Commerce Home Page- When WooCommerce is installed', () => {
 	} );
 
 	it( 'Verify next step "Add a Product"', () => {
-		cy.contains( 'Add a product' ).as( 'addProduct' ).should( 'exist' );
-		cy.get( '@addProduct' ).click();
+		waitForNextSteps();
+		cy.contains( '.nfd-grid.nfd-gap-4 ul li a', 'Add a product', {
```
Comment (aratidgr8) [inline tests/cypress/integration/Home/homePageWithWoo.cy.js:200]: We were going to stop checking texts I think?

Comment (sangeetha-nayak) [reply tests/cypress/integration/Home/homePageWithWoo.cy.js:200]: In un-avoidable cases where text is the only unique locator we have still kept text identifier. For example **Next Steps** list elements (<li>) which are all having same html properties except the text.

Comment (aratidgr8) [reply tests/cypress/integration/Home/homePageWithWoo.cy.js:200]: In that case I think we should check for pt_BR as well for these texts in un-avoidable cases

Comment (sangeetha-nayak) [reply tests/cypress/integration/Home/homePageWithWoo.cy.js:200]: Currently we are not running tests against pt_BR. In future yes, we must add some unique identifiers like id, data-test-id etc to such elements and make them independent of text.

**Pull request merged**